examples/tokens/ERC1155ownable.vy

```diff
@@ -0,0 +1,363 @@
+# @version >=0.3.1
+
+# @dev Implementation of ERC-1155 non-fungible token standard
+# @dev depends on Dynarray and bytes4, which will be available with 0.3.2 release
```

Comment:
```suggestion
```
(this line is redundant with the update to the version pragma above)

Reply:
Deleted!